Source/DafnyDriver/DafnyDriver.cs

```diff
@@ -23,7 +23,7 @@ namespace Microsoft.Dafny
 
   public class DafnyDriver
   {
-    public enum ExitValue { VERIFIED = 0, PREPROCESSING_ERROR, DAFNY_ERROR, COMPILE_ERROR, NOT_VERIFIED }
+    public enum ExitValue { VERIFIED = 0, PREPROCESSING_ERROR, DAFNY_ERROR, COMPILE_ERROR, NOT_VERIFIED, DONT_PROCESS_FILES }
```

Comment:
It's unfortunate this becomes the exit value of the whole `dafny` CLI and therefore looks like failure. Looks like it's already a problem for a non-error condition like `NOT_VERIFIED` though. You might just want to add a TODO.

Also (this can be done next PR) I noticed that `dafny /help` also prints out `*** Error: No input files were specified.`. Can we apply the same fix there?

Reply:
That makes sense. I'll add a TODO for refactoring the non-error exit values, and open issues for both that and for suppressing the error message during `dafny /help`.

Reply:
On second thought, perhaps `NOT_VERIFIED` should be considered an error/non-zero condition, in the sense that the user should treat verification just as they treat type-checking and compilation - if it fails, the code's not good. If that makes sense, I'll just change the behavior of `/help` and `/version` to exit without processing files, and with exit code 0.

Reply:
Aha, my mistake, I misread `NOT_VERIFIED` as "verification skipped" (such as when using the `/noVerify` switch) rather than "verification failed".

With that in mind, can you do a follow-up PR right away (since this is already merged) to:

1. remove `DONT_PROCESS_FILES`
2. use `VERIFIED` instead, and
3. rename `NOT_VERIFIED` -> `VERIFICATION_ERROR` and `VERIFIED` -> `SUCCESS`

Reply:
As it stands that doesn't quite seem like that'll work, since ExitValue is used both by ProcessCommandLineArguments and by ThreadMain to indicate different things. (In particular, when ProcessCommandLineArguments returns VERIFIED, ThreadMain will attempt to actually run the verification...) I'll refactor the former to use a return type that better fits its behavior.